src/components/NavigationBar.jsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import url from 'url';
+import {Container, Nav, Navbar} from "react-bootstrap";
+import {Link, NavLink} from "react-router-dom";
+import "./styles/NavigationBar.css";
+
+const logo = url.resolve(process.env.PUBLIC_URL, "/icons/se-logo-trimmed.png");
+
+const NavigationBar = () => (
+  <Navbar bg="transparent" expand="sm" collapseOnSelect >
```

Comment:
Instead of setting `bg="transparent"` and laying it over the jumbotron, we could try doing: `style={{backgroundColor: "#F0EDF1"}}` which is the colour of the jumbotron and I can make sure the hero fits within bounds of the jumbotron.

Reply:
Feel free to make a variable in _colors.scss & the jsx files called `hero-bkg-colour`, `var(--hero-bkg-color)` and update all instances.